tailwind.config.js

```diff
@@ -75,12 +81,27 @@ module.exports = {
     },
   },
 
+  variants: {
+    extend: {
+      animation: ['motion-safe'],
```

Comment:
This ensures we can build accessible animations and respect browsers that have them disabled.